images/kubeflow/README.md

```diff
@@ -0,0 +1,60 @@
+<!--monopod:start-->
+# kubeflow
+| | |
+| - | - |
+| **Status** | stable |
+| **OCI Reference** | `cgr.dev/chainguard/kubeflow` |
```

Comment:
I think this is the issue with `images/kubeflow` representing two repos.

Reply:
Three actually; `kubeflow/kubeflow`, `kubeflow/katib`, and `kubeflow/pipelines`.